### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [Homebrew/discussions#5456] [Linux] [answered]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 43eaeca50fe3b6c755f3fc9bc6f43669e0db1039
Last commit: 9 hours ago
Core tap JSON: 24 Jun 16:46 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: octa-core 64-bit skylake
Clang: N/A
Git: 2.39.2 => /bin/git
Curl: 7.88.1 => /bin/curl
Kernel: Linux 6.1.0-17-amd64 x86_64 GNU/Linux
OS: Debian GNU/Linux 12 (bookworm) (bookworm)
Host glibc: 2.36
/usr/bin/gcc: N/A
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: 14.1.0_1
xorg: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: No developer tools installed.
Install Clang or run `brew install gcc`.
```

### Description of issue

![image](https://github.com/Homebrew/discussions/assets/103120356/d55942d4-5973-47a9-8e84-d839b03c12e8)

**Comment by gromgit** [comment]
What's the output of `brew ls bottom`? Hint: the program name you're looking for is _not_ `bottom`.

**Reply by pizzav-xyz** [reply]
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/.crates.toml
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/.crates2.json
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/bin/btm
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/etc/bash_completion.d/btm.bash
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/sbom.spdx.json
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/share/fish/vendor_completions.d/btm.fish
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/share/man/man1/btm.1
/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/share/zsh/site-functions/_btm

**Reply by gromgit** [reply, marked as answer]
> `/home/linuxbrew/.linuxbrew/Cellar/bottom/0.9.6/bin/btm`

As that line says, you want to run `btm`, not `bottom`. This is also [documented in the software's website](https://clementtsang.github.io/bottom/0.9.6/usage/general-usage/).
